appveyor.yml

```diff
@@ -1,20 +1,33 @@
 environment:
   SAXON_CP: '%TEMP%\xspec\saxon\saxon9he.jar'
   matrix:
-    # latest Saxon 9.7 version
+    # latest Saxon 9.7 version and XML Calabash
     - SAXON_VERSION: 9.7.0-15
+      XMLCALABASH_VERSION: 1.1.15-97

```

Comment:
At the moment the [.travis](https://github.com/xspec/xspec/blob/master/.travis.yml) runs XProc for all three flavours of Saxon. In the AppVeyor configuration the test for XProc is only executed with Saxon 9.7.0-15. You can verify this by comparing the [failed build for AppVeyor](https://ci.appveyor.com/project/AirQuick/xspec/build/85-win_xproc) (1 build failed) and the [failed build for Travis](https://travis-ci.org/AirQuick/xspec/builds/206112024) (3 builds failed). 

Shall we align the AppVeyor configuration with the Travis configuration? How complicated is it to run the test for all three versions of Saxon with AppVeyor?